<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Thread Pool on mojidong blog</title>
    <link>https://www.mojidong.com/tags/thread-pool/</link>
    <description>Recent content in Thread Pool on mojidong blog</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>zh-cn</language>
    <lastBuildDate>Sat, 02 Mar 2019 20:58:56 +0800</lastBuildDate>
    
	<atom:link href="https://www.mojidong.com/tags/thread-pool/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>thread pool中使用thread local的问题</title>
      <link>https://www.mojidong.com/post/2019-03-02-thread-pool/</link>
      <pubDate>Sat, 02 Mar 2019 20:58:56 +0800</pubDate>
      
      <guid>https://www.mojidong.com/post/2019-03-02-thread-pool/</guid>
      <description>最近发现生产环境一个十分诡异的问题，这里与大家分享一下。 问题 最近发现线上服务报错，报错是偶发的，会自动恢复。主体代码如下: 1 2 3 4 5 6 7 8 9</description>
    </item>
    
  </channel>
</rss>